datafusion/core/tests/sqllogictests/test_files/timestamps.slt

```diff
@@ -500,6 +500,118 @@ FROM (
     (TIMESTAMP '2021-06-10 17:19:10Z', TIMESTAMP '2001-01-01T00:00:00Z', 0.3)
   ) as t (time, origin, val)
 
+# MONTH/YEAR INTERVAL ON ARRAY DATA STILL NOT PROVIDE CORRECT ANSWERS YET
```

Comment:
Test on array data (equivalent to testing on table data). These tests do not yet return the correct answers. See below for my comment on how to fix this in my next PR